src/client/pages/OfferNew/Perils/PerilModal.tsx

```diff
@@ -19,235 +22,105 @@ interface PerilModalProps {
   setCurrentPeril: (perilIndex: number) => void
 }
 
-const Header = styled('div')`
-  width: 100%;
-  height: 8.75rem;
-  background-color: ${colorsV3.white};
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: flex-end;
-  position: relative;
-  border-bottom: 1px solid rgba(0, 0, 0, 0.1);
-  margin-bottom: 0.125rem;
+const ModalWrapper = styled(Modal)`
+  max-width: 33rem;
```

Comment:
I'm thinking we should let the Modal be equal in height between Perils. Otherwise it becomes a bit hard to navigate when the arrow buttons move between Perils :) Its maybe a bit more complex compared to market-web since the Modal is used more, so changing the height for non dynamic modal might break for some implementations. So you could add it here :D  (42 rem value is taken from the PerilModal in market-web) 
```suggestion
  max-width: 33rem;
  height: 100%;
  max-height: 42rem;
```

https://user-images.githubusercontent.com/6661511/170005156-d0e81f53-5418-47e6-8816-d3f8f4218d25.mov